port/page-permissions/model.go

```diff
@@ -0,0 +1,15 @@
+package page_permissions
+
+import "github.com/hashicorp/terraform-plugin-framework/types"
+
+type ReadPagePermissionsModel struct {
```

Comment:
you don't support the good 'ol `Give access to everyone in the organization` button?